scripts/lib/install

```diff
@@ -600,10 +708,45 @@ database user, and then run:
 
    su zulip -c '/home/zulip/deployments/current/scripts/setup/initialize-database'
    su zulip -c '/home/zulip/deployments/current/manage.py generate_realm_creation_link'
+
+If you specified you want to enable the push notifications service, you'll also need to
+manually register your server by running:
+
+   su zulip -c '/home/zulip/deployments/current/manage.py register_server'
```

Comment:
We know if they did, and an `if` statement seems pretty easy to toss in here.

Reply:
oh yeah, added:
```
    if [ -n "$PUSH_NOTIFICATIONS" ]; then
        echo
        echo "Since you specified you want to enable the push notifications service, you'll also need to "
        echo "manually register your server by running:"
        echo
        echo "   su zulip -c '/home/zulip/deployments/current/manage.py register_server'"
        echo
    fi
    exit 0
```